### PR TITLE
Adding Jira Dashboard plugin to the Directory

### PR DESCRIPTION
**Pull request by fridajac**
Added the Jira Dashboard plugin to the Directory.

## Hey, I just made a Pull Request!

The Jira Dashboard plugin is now open sourced and available. The plugin has been added to the plugin Directory at backstage.io. 

#### :heavy_check_mark: Checklist

- [ ] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [ ] Added or updated documentation
- [ ] Tests for new functionality and regression tests for bug fixes
- [ ] Screenshots attached (for UI changes)
- [x] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

File: microsite/data/plugins/jira-dashboard [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+title: Jira Dashboard
+author: AxisCommunications
+authorUrl: https://github.com/AxisCommunications
+category: Agile Planning
+description: Allows you to fetch and display Jira issues for your entity. You get quickly access to issue summaries to achieve better task visibility and more efficient project management. 
+documentation: https://github.com/AxisCommunications/backstage-plugins
```
Comment (awanlin) [inline microsite/data/plugins/jira-dashboard:7]: You'll want to link them to the specific documentation for this plugin, I would use this: https://github.com/AxisCommunications/backstage-plugins/blob/main/plugins/jira-dashboard/README.md

File: microsite/data/plugins/jira-dashboard [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+iconUrl: https://github.com/AxisCommunications/backstage-plugins/blob/main/plugins/jira-dashboard/media/jira-logo.png
```
Comment (awanlin) [inline microsite/data/plugins/jira-dashboard:8]: This looks to be an official Atlassian Jira logo, can you confirm we have permissions to use this?

**Review by awanlin — CHANGES_REQUESTED** [review]
Very excited to see this get into the Plugin Directory @fridajac! Left a pair of comments

File: microsite/data/plugins/jira-dashboard (hunk 2 above)
Comment (awanlin) [reply microsite/data/plugins/jira-dashboard:8]: Looks like perhaps you can: https://www.atlassian.com/legal/trademark#:~:text=You%20may%20use%20the%20Atlassian,transposing%20Atlassian's%20logo%20with%20your

Comment (awanlin) [reply microsite/data/plugins/jira-dashboard:8]: @benjdlambert what's your thoughts?

**Review by awanlin — APPROVED** [review]
🚀

**Review by tudi2d — APPROVED** [review]
:shipit:

**Pull request merged**